src/services/posts/get-post.test.js

```diff
@@ -0,0 +1,140 @@
+const {expect} = require('chai')
+const getPost = require("./get-post")
+const {fixtures} = require('../../../test/utils')
+
+
+describe('Get POST ', () => {
+
+	let user
+
+	before(async ()=>{
+		user = await fixtures.insertUser() 
+	})
+
+	describe('Gif', () => {
+		
+		let post
+		let comment
+
+		before(async ()=>{
+			post = await fixtures.insertPost({
+				userId : user.id , 
+				type : 'gif'
+			})
+			comment = await fixtures.insertPostComment({
+				id : post.id,
+				userId : post.userId,
+				type: 'gif'
```

Comment:
i don't think this function takes a `type` parameter

Reply:
yes please